index.html

```diff
@@ -94,7 +97,55 @@ <h3>Multi-Post Stories</h3>
+                        <div class="technologies">
+                            <ul>
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby on rails</li>
+                            </ul>
+                        </div>
+                        <div class="modal-body">
+                            <div class="description-modal">
+                                <img src="./Images/Snapshoot.svg"
+                                    alt="Keeping track of hundreds of components website image">
+                                <div>
+                                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+                                        Ipsum has been the industry's standard dummy text ever since the 1500s, when an
+                                        unknown printer took a galley of type and scrambled it 1960s with the releaLorem
+                                        Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
+                                        has
+                                        been the industry's standard dummy text ever since the 1500s, when an unknown
+                                        printer took a galley of type and scrambled it 1960s with the releax map lapora
+                                        verita.</p>
+                                    <div class="footer-popup">
+                                        <button class="btn btn-style iconsbutton" type="button"
+                                            onclick="window.open('https://regiss05.github.io/Portfolio-setup-and-mobile-version-skeleton')">
+                                            <span>See Live</span>
+                                            <span class="icon-btn">
+                                                <ion-icon name="pulse-outline"></ion-icon>
+                                            </span>
+                                        </button>
+                                        <button class="btn btn-style iconsbutton" type="button"
+                                            onclick="window.open('https://github.com/Regiss05/Portfolio-setup-and-mobile-version-skeleton')">
+                                            <span>See Source</span>
+                                            <span class="icon-btn">
+                                                <ion-icon name="logo-github"></ion-icon>
+                                            </span>
+                                        </button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div id="overlay"></div>
                 </div>
             </section>
             <div class="container-form">
```

Comment:
- [ ] Here happens the same as I mentioned before, you now are supposed to store all your elements of this `div` inside a JavaScript object and make the elements appear dynamically once the page loads, so try to apply the changes here as well.